File: Practice/Tutorials/30_Days_of_Code/Day_06_-_Let_s_Review.php
<?php
	$_fp = fopen("php://stdin", "r");
	/* Enter your code here. Read input from STDIN. Print output to STDOUT */

	$testCases = fgets($_fp);

	for ($i = 0; $i < $testCases; $i++) {
		$lineText = trim(fgets($_fp));
		$lineTextArray = str_split($lineText, 1);
		
		for ($j = 0; $j < count($lineTextArray); $j = $j + 2) {
			print $lineTextArray[$j];
		}
		
		print " ";
		
		for ($j = 1; $j < count($lineTextArray); $j = $j + 2) {
			print $lineTextArray[$j];
		}
		
		print "\n";
	}
?>

File: Practice/Tutorials/30_Days_of_Code/Day_06_-_Let_s_Review.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args)
    {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner input = new Scanner(System.in);
        int numberOfInputsLines = Integer.parseInt(input.nextLine());

        for (int i = 0; i < numberOfInputsLines; i++)
        {
            String lineText = input.nextLine();
            
            String[] lineTextArray = lineText.split("");
            for (int j = 0; j < lineTextArray.length; j = j + 2)
            {
                System.out.print(lineTextArray[j]);
            }

            System.out.print(" ");

            for (int j = 1; j < lineTextArray.length; j = j + 2)
            {
                System.out.print(lineTextArray[j]);
            }

            System.out.println();
        }
    }
}
